src/Builders/EmailBuilder.cs

```diff
@@ -19,18 +19,35 @@ private static MimeMessage BuildMessage(EmailMessage emailMessage)
             var toEmails = emailMessage.ToEmail.Split(new string[] { "," }, StringSplitOptions.RemoveEmptyEntries);
 
             foreach (var email in toEmails)
```

Comment:
Think about the other code and the different ways currently implemented. I'd say go with curlies in this instance. But happy for you to make the call.